### ISSUE TITLE
Make focus indicator more visible on black buttons

### ISSUE THREAD

**Issue by EmmaDawsonDev** [conermurphy/conermurphy.com#223]
The focus indicator is black, which means it's very hard to spot when the black buttons are in focus. Consider changing the color so it has at least a 3:1 contrast ratio to both the page background and the button background or offset the outline slightly so there's a space between the buttons and the focus indicator.
